src/components/EmptyState/EmptyState.tsx

```diff
@@ -25,6 +25,8 @@ export interface Props {
   action: Action;
   /** Secondary action for empty state */
   secondaryAction?: Action;
+  /** Secondary elements to display inside empty state */
```

Comment:
Maybe mention the elements will be displayed below the actions?

Reply:
Secondary empty state suggestion? 

I added some other thoughts on the PR as well